openid-sse-framework-1_0.xml

```diff
@@ -852,19 +633,125 @@ Cache-Control: no-store
           <t>TODO: consider adding a IANA Registry for stream configuration metadata, similar to Section 7.1.1 of
             <xref target="OAUTH-DISCOVERY"/>. This would allow other specs to add to the stream configuration.</t>
 
-          <section anchor="reading-a-streams-configuration"
-            title="Reading a Stream’s Configuration">
-            <t>An Event Receiver gets the current configuration of a stream by making an HTTP GET
+          <section anchor="creating-a-stream" title="Creating a Stream">
+            <t>In order to communicate events from a Transmitter to a Receiver, a Receiver
+              MUST first create an Event Stream. An Event Receiver creates a stream by making an HTTP POST
               request to the Configuration Endpoint. On receiving a valid request the Event
               Transmitter responds with a <spanx style="verb">200 OK</spanx> response containing a <xref target="RFC7159">JSON</xref>
               representation of the stream’s configuration in the body.</t>
 
-            <t>The following is a non-normative example request to read an Event Stream’s
-              configuration:</t>
+            <t>The HTTP POST request MAY contain the read/write values of the <xref target="stream-config">Stream Configuration</xref>
```

Comment:
"read/write values" may be confusing. Since this is a create w/ POST, can we just say "values"?

Reply:
I have always found the Read-Only vs Read-Write distinction of these values to be confusing. Would it make more sense to call them Transmitter-Supplied and Client-Supplied values?

Reply:
Changes in [0f4f772](https://github.com/openid/sse/pull/9/commits/0f4f772db46c14489dac14dd2d2717be3f6c3ff1) look good to me!